packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -30,11 +30,15 @@ import {
 declare const Zone: ZoneType & { prototype: Zone };
 
 // Delay of 50 ms to reset currentEventTracingZone.
-const RESET_TRACING_ZONE_DELAY = 50;
+export const RESET_TRACING_ZONE_DELAY = 50;
 
 export class InteractionTracker {
   // Allows to track several events triggered by the same user interaction in the right Zone.
-  private currentEventTracingZone?: Zone;
+  currentEventTracingZone?: Zone;
```

Comment:
It's necessary to test when there are overlapping interactions and check the interactions have different ID. Also, when there are several event clicks corresponding to the same interaction, we want to make sure there is not a new zone for those click events. I am open to receive recommendations on a way to avoid exposing this.

Reply:
Update: I stopped exposing this as final decision was to leave this kind of testing of a future refactoring on `InteractionTracker`